tika/server.go

```diff
@@ -39,6 +40,41 @@ type Server struct {
 	url  string // url is derived from port.
 	port string
 	cmd  *exec.Cmd
+	child *ChildOptions
+}
+
+// Command line parameters that can be used when tika is run with the -spawnChild option.
+// If field is 0 associated flag is not included.
+type ChildOptions struct {
+	MaxFiles int
+	TaskPulseMillis int
+	TaskTimeoutMillis int
+	PingPulseMillis int
+	PingTimeoutMillis int
+}
+
+func (co *ChildOptions) getArgs() []string {
+	if co == nil {
+		return []string{}
```

Comment:
In general, should avoid making a distinction between `nil` and an empty slice. So, it's most common to just return `nil`.